### PR TITLE
Update event schema version type

### PR DESCRIPTION
**Pull request by davidbrochart**

### CODE REVIEW

**Comment by davidbrochart** [comment]
It looks like UI tests are flaky again @trungleduc?
For instance:
- actual:
image
- expected:
image
- diff:
image

**Comment by trungleduc** [comment]
yeah, your only choice is to relax the diff ratio

File: ui-tests/playwright.config.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = {
   },
   expect: {
     toMatchSnapshot: {
-      maxDiffPixelRatio: 0.0001
+      maxDiffPixelRatio: 0.01
```
Comment (trungleduc) [inline ui-tests/playwright.config.js:23]:
```suggestion
      maxDiffPixelRatio: 0.001
```
1% might be overkill

Comment (davidbrochart) [reply ui-tests/playwright.config.js]: It was not, I'm reverting that change.

**Comment by davidbrochart** [comment]
Ubuntu test failures are tracked in #416.

**Comment by krassowski** [comment]
> Ubuntu test failures are tracked in https://github.com/jupyterlab/jupyter-collaboration/issues/416.

Should be fixed in 15 minutes with release of 4.3.4 :crossed_fingers:

**Comment by davidbrochart** [comment]
Let's see if it picks up JupyterLab v4.3.4.

**Comment by davidbrochart** [comment]
CI is green again :tada:

**Comment by brichet** [comment]
@meeseeksdev please backport to 3.x

**Pull request merged**